src/cockpit/packages.py

```diff
@@ -298,3 +307,19 @@ def get_bridges(self):
         for package in sorted(self.packages.values(), key=lambda package: -package.priority):
             bridges.extend(package.bridges)
         return bridges
+
+    @bus.Interface.Method()
+    def reload(self):
+        self.packages = {}
+        self.load_packages()
```

Comment:
I guess this is the part that will have to get more intelligent in the future if we want to support the checksums properly, right?  We need to be able to detect the case if the scanned package is the same as the package we had before, on a package-by-package basis.

There's code in `load_packages()` that supports this — the meaning of passing the list of checksum objects was that multiple checksums (global and local) could be built.  What's missing is the part to record the individual checksum of each package...

I guess it's a nice chance for a follow-up, anyway.  I question if we want this stuff at all in the future, but at least implementing it shouldn't be too much more work at this point.

Reply:
> I guess this is the part that will have to get more intelligent in the future if we want to support the checksums properly, right? We need to be able to detect the case if the scanned package is the same as the package we had before, on a package-by-package basis.

Yes, exactly.

> I guess it's a nice chance for a follow-up, anyway. I question if we want this stuff at all in the future, but at least implementing it shouldn't be too much more work at this point.

Yeah, it's not a lot of work, but it's a complicated scheme and it would simplify things in general if we could get rid of it.